cli/src/pcluster/validators/slurm_settings_validator.py

```diff
@@ -182,3 +182,19 @@ def _validate(self, queue_name: str, compute_resources: List[Dict]):
                 f"({max_static}): {bad_dynamic_priorities}.",
                 FailureLevel.WARNING,
             )
+
+
+class ExternalSlurmdbdVsDatabaseIncompatibility(Validator):
+    """
+    External Slurmdbd vs Database Validator.
+
+    This validator checks that ExternalSlurmdbd and Database are not defined at the same time in the cluster
+    configuration within SlurmSettings.
+    """
+
+    def _validate(self, database, external_slurmdbd):
+        if (database is not None) and (external_slurmdbd is not None):
+            self._add_failure(
+                "Database and ExternalSlurmdbd cannot be defined at the same time within SlurmSettings.",
```

Comment:
Has the PM reviewed this message ?

Reply:
No, but this is a temporary message, I will review all of this later with the PM.